health/checks/checks.go

```diff
@@ -33,3 +41,122 @@ func HTTPChecker(r string) health.Checker {
 		return nil
 	})
 }
+
+// PingChecker sends an ICMP Echo request and verifies that a
+// response was reveived, and within the maximum round-trip time,
+// taking the application out of rotation otherwise
+func PingChecker(a string, maxRttMsec int) health.Checker {
```

Comment:
Please don't use untyped arguments for time durations. Use `time.Duration` and call it `maxRTT`. Values can be specified like `10*time.Millisecond` and are type safe.